CODEOWNERS

```
@@ -95,6 +95,7 @@ k8s/**/common/sscs/namespace.yaml @hmcts/sscs
 k8s/**/common/sscs/identity.yaml @hmcts/sscs
 k8s/**/common/sscs/sscs-tribunals-api.yaml @hmcts/sscs
 k8s/**/common/sscs/sscs-evidence-share.yaml @hmcts/sscs
+k8s/**/common/sscs/sscs-tya-notif.yaml @hmcts/sscs
```

this should just be  aat until you have  approval from secops for prod

```suggestion
k8s/aat/common/sscs/sscs-tya-notif.yaml @hmcts/sscs
```

---

Thanks Tim, done.